include/beast/core/detail/is_call_possible.hpp

```diff
@@ -13,41 +13,6 @@
 namespace beast {
```

Comment:
Whoops... yes. Its probably not even included anywhere anymore. Or at least, it shouldn't be.
willfix